src/render/portable/utils.rs

```diff
@@ -73,11 +74,23 @@ pub(crate) fn render_static_files<P: AsRef<Path>>(path: P) -> Result<()> {
 
     for (name, file) in files {
         let mut out = File::create(path.join(Path::new(name)))?;
-        out.write_all(file.as_bytes())?;
+        let minified = minify_js(file)?;
+        out.write_all(&minified)?;
```

Comment:
I think all files written here are already minified except the one css file that is not big at all and already included in each report one time.

```suggestion
        out.write_all(file.as_bytes())?;
```